### ISSUE TITLE
Don't fold diffs when staging changes

### ISSUE THREAD

**Issue by Wilfred** [ananthakumaran/monky#61]
If I press `<TAB> s`, monky expands the diff, but unexpands when the file is staged. This means I need to expand the diff again when writing the commit message.

I know I've been filing several issues, I'm happy to send a PR if you're interested :)

**Comment by Bounga** [comment]
@Wilfred Yes please do send PRs for your various issues. I'll be happy to review, test and hopefully merge them if everything is OK.
